### ISSUE TITLE
How can I edit the default ignored files list and sync ALL files?

### ISSUE THREAD

**Issue by matteoipri** [owncloud/client#5310]
Hi all,
I installed ownCloud on an x64 server running Arch Linux with nginx, mariadb and php 7.
After trying it out a little I installed the Desktop client on my Mac.
I put some folders in the owncloud sync folder and then noticed that several files were not synced, like `.htaccess` and `filename~` files.
I need those files synced, so I tried to edit the list of excluded files but I found a long list of patterns that I can not edit nor delete.
Looking in the github repo of the client I found the "sync-exclude.lst" file and I edited it on my PC, but I know it will be overwritten with the next update of the client app. Now most of my files get synced.
Why are all those files excluded?
Why are `.htaccess` not synced with an error related to not available space on my server? (other new files get synced regularly)
Thanks,
Matteo

### Expected behaviour
I want ALL files to be synced and to control what is excluded.

### Actual behaviour
Several file types are not synced, some because of the settings, some because of some server errors.

### Steps to reproduce
1. Put folders and files in the sync folder
2. Look at the `Not Synced` pane in the settings of the client

### Server configuration
Operating system:
Arch Linux
Web server: 
nginx
Database:
mariadb
PHP version:
7
ownCloud version:
9.1.1

### Client configuration
Client version:
2.2.4
Operating system:
OS X 10.10.5
OS language:
English
Installation path of client:
/Applications

**Comment by RealRancor** [comment]
@matteoipri Please note that this is a bugtracker for the sync client. Support questions like this needs to go to https://central.owncloud.org/

Short answer to the .htaccess: They are blocked by the oC server for security reasons. You might find some additional info about that when searching at https://github.com/owncloud/core/issues

**Comment by matteoipri** [comment]
@RealRancor Thanks! I've found the issues you're talking about.
Still I think the user should be able to edit the default sync-exclude list in the client.
I understand the security issues related to the `.htaccess` files, but, as another user pointed out, advanced users should be able to make the decision themselves. For example, I'm running a LEMP stack, thus htaccess files are not a concern for me and I should be able to enable the syncing of those file in my client. So, if the server I use is administered by an admin that knows how to handle htaccess files, my client will sync them without any problem. More over, as a user, I can be interested in having also backup files (I mean the one ending with a tilde) synced and versioned. This last again, is a client issue. Am I wrong? Correct me, please, if so :)

**Comment by RealRancor** [comment]
> @matteoipri Please note that this is a bugtracker for the sync client. Support questions like this needs to go to https://central.owncloud.org/

**Comment by RealRancor** [comment]
Btw. the request to edit the sync-exclude list from the client is a duplicate of https://github.com/owncloud/client/issues/2816

**Comment by matteoipri** [comment]
Ok, thanks, I'll close this issue and comment on the other.
